Added `PyErr::write_unraisable()` to report an unraisable exception to Python.
